src/ucp/api/ucp_def.h

```diff
@@ -731,6 +731,13 @@ typedef struct ucp_ep_params {
      */
     ucs_sock_addr_t         sockaddr;
 
+    /**
+     * Local address in the form of a sockaddr; this field should be set
+     * along with its corresponding bit in the field_mask - @ref
+     * UCP_EP_PARAM_FIELD_LOCAL_ADDR.
```

Comment:
pls add "Forces using a specific network device for the new connection"

Reply:
fixed